### PR TITLE
Added CagedMobs to supported mods list

### PR DESCRIPTION
**Pull request by Corgam**

### CODE REVIEW

File: docs/mods/CagedMobs/CagedMobs.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+```
+
+Tier:
+
+```zenscript
+    // To change required tier (1,2 or 3)
+    cow.setTier(2);
+```
+
+Waterlogging requirement:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:112]:
```suggestion
### Waterlogging requirement:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+With it you can add a new additionalLoot recipe (it won't have any items yet, you need to add them later):
+
+```zenscript
+    // New additional loot:  create(id, entityType).
+    // The id string cannot contain any capital letters.
+    val leather = additionalLoots.create("exampledatapack:leatherfrompig",<entitytype:minecraft:pig>);
+```
+
+or you can get one of the already existing ones with **getAdditionalLoot(id)** methode:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:144]:
```suggestion
or you can get one of the already existing ones with the `getAdditionalLoot(id)` method:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+
+```zenscript
+    val rose = additionalLoots.getAdditionalLoot("cagedmobs:additional_loot/wither_rose");
+```
+
+### Modifying the additional loot:
+
+When you have the **additionalLoot value** you can modify its drops and entity type in the same way as in the entity recipes (with functions: addLoot(), clearLoot(), removeLoot() and setEntityType()).
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:152]:
```suggestion
When you have the `additionalLoot value` you can modify its drops and entity type in the same way as with the entity recipes (with functions: `addLoot()`, `clearLoot()`, `removeLoot()` and `setEntityType()`).
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+# **Environments**
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:154]:
```suggestion
## Environments
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    val envs = <recipetype:cagedmobs:env_data>;
+```
+
+### Getting the environment value:
+
+Then you can either create a new environment or modify an old one:
+
+```zenscript
+    // New environment with one tag:  create(id, inputItem, renderBlock, growthModifier, tags)
+    //The id string cannot contain any capital letters.
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:175]:
```suggestion
    //The id string cannot contain any capital letters or spaces.
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+    val forest = envs.create("exampledatapack:forest",<item:minecraft:grass_block>, <blockstate:minecraft:grass_block>, 1.0, ["forest"]);
+
+    // New environment with multiple tags:
+    //The id string cannot contain any capital letters.
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:179]:
```suggestion
    //The id string cannot contain any capital letters or spaces.
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+    val home = envs.create("exampledatapack:home",<item:minecraft:oak_log>, <blockstate:minecraft:oak_planks>, 1.0, ["home","house"]);
+
+    // Get an old environment:
+    val farm = envs.getEnvironment("cagedmobs:envs/farm");
+```
+
+Both ways will give you an **Environment value**, which you can use to do further changes.
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:186]:
```suggestion
Both methods will give you an `Environment value`, which you can use to do further changes.
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+### Modifying the environment:
+
+With your **environment value** you can use several methodes:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:190]:
```suggestion
With your `environment value` you can use several methods:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+Categories:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:192]:
```suggestion
### Categories:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+```zenscript
+// To remove just one category (tag):
+home.removeCategory("home");
+// To clear all categories (tags):
+home.clearCategories();
+// To add a new category (tag):
+home.addCategory("building");
+```
+
+Input item:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:203]:
```suggestion
### Input item:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+
+```zenscript
+// To change the input item:
+home.setInputItem(<item:minecraft:spruce_planks>);
+```
+
+Display block:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:210]:
```suggestion
### Display block:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+
+```zenscript
+// To change the displayed block:
+home.setDisplay(<blockstate:minecraft:birch_planks>);
+```
+
+Growth modifier:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:217]:
```suggestion
### Growth modifier:
```

File: docs/mods/CagedMobs/CagedMobs.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+### Importing the manager:
+
+First of all, you will need a entity manager. You can import it into your script with lines below:
+
+```zenscript
+    import mods.cagedmobs.EntitiesManager;
+    val entities = <recipetype:cagedmobs:mob_data>;
+```
+
+### Adding new entity:
```
Comment (jaredlll08) [inline docs/mods/CagedMobs/CagedMobs.md:25]:
```suggestion
### Adding new entities:
```

**Review by jaredlll08 — CHANGES_REQUESTED** [review]

**Comment by jaredlll08** [comment]
You also need to add your page to the docs.json file
https://github.com/CraftTweaker/CraftTweaker-Documentation/blob/1.16/docs.json

**Review by Corgam — COMMENTED** [review]
Changed the docs.json file

**Comment by Corgam** [comment]
This is my first time using Pull requests, I hope I did everything alright

**Pull request merged**